### PR TITLE
Improved logging time taken message showing milliseconds

### PR DESCRIPTION
**Pull request by vfdev-5**
Description:
- Improved logging time taken message showing milliseconds in Engine, utils and Cifar10 example
- removed eval pbar from cifar10

Check list:

- [ ] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [ ] Documentation is updated (if required)

### CODE REVIEW

File: examples/contrib/cifar10/main.py [hunk 1]
```diff
@@ -380,9 +380,6 @@ def evaluate_step(engine: Engine, batch):
     for name, metric in metrics.items():
         metric.attach(evaluator, name)
 
-    if idist.get_rank() == 0 and (not config["with_clearml"]):
```
Comment (sadra-barikbin) [inline examples/contrib/cifar10/main.py:383]: Why did you this pbar?

Comment (vfdev-5) [reply examples/contrib/cifar10/main.py:383]: I does not look helpful and also adds useless lines if logs are written into a file

File: ignite/engine/engine.py [hunk 2]
```diff
@@ -776,7 +776,7 @@ def _internal_run(self) -> State:
             # update time wrt handlers
             self.state.times[Events.COMPLETED.name] = time_taken
             hours, mins, secs = _to_hours_mins_secs(time_taken)
-            self.logger.info(f"Engine run complete. Time taken: {hours:02d}:{mins:02d}:{secs:02d}")
+            self.logger.info(f"Engine run complete. Time taken: {hours:02d}:{mins:02d}:{secs:06.3f}")
```
Comment (sadra-barikbin) [inline ignite/engine/engine.py:779]: I do not get `6` in `06.3f`. seconds does not outnumber 60 which has two digits. why have you assumed 6 digits for it?

Comment (vfdev-5) [reply ignite/engine/engine.py:779]: Yes, `06.3f` means 6 digits in total for float number of seconds. Check for example:
```python
x = 7.890
print(f"{x:06.3f}")
```
>
```
07.890
```

I need the first padding zero to have uniform representation + milliseconds

File: examples/contrib/cifar10/main.py [hunk 3]
```diff
@@ -260,7 +260,7 @@ def initialize(config):
 
 def log_metrics(logger, epoch, elapsed, tag, metrics):
     metrics_output = "\n".join([f"\t{k}: {v}" for k, v in metrics.items()])
-    logger.info(f"\nEpoch {epoch} - Evaluation time (seconds): {elapsed:.2f} - {tag} metrics:\n {metrics_output}")
+    logger.info(f"Epoch[{epoch}] - Evaluation time (seconds): {elapsed:.3f}\n - {tag} metrics:\n {metrics_output}")
```
Comment (sadra-barikbin) [inline examples/contrib/cifar10/main.py:263]: If evaluation time is long, isn't more understandable to have:
```suggestion
    minutes, seconds = divmod(elapsed, 60)
    logger.info(f"Epoch[{epoch}] - Evaluation time : {int(minutes)} min and {int(seconds)} s\n - {tag} metrics:\n {metrics_output}")
```

**Review by sadra-barikbin — APPROVED** [review]

File: examples/contrib/cifar10/main.py (hunk 3 above)
Comment (vfdev-5) [reply examples/contrib/cifar10/main.py:263]: For cifar10 it wont be that long. It is around 2-3 seconds for resnet18. But in any case `elapsed` refers to `engine.state.times[Events.COMPLETED]` and where the time is seconds no matter how much time it took.

**Pull request merged**